### ISSUE TITLE
Error with version 1.1.5

### ISSUE THREAD

**Issue by aurbano** [gor181/react-notification-system-redux#48]
I think there's an issue with version 1.1.5, seems related to [this comment on another issue](https://github.com/gor181/react-notification-system-redux/issues/28#issuecomment-297411658).

```
TypeError: _reactNotificationSystemRedux2.default.success is not a function
```

**Comment by mutsumiya** [comment]
I got the same error but for success and error. And then fixed by changing from:

import Notifications from 'react-notification-system-redux'
to

import * as Notifications from 'react-notification-system-redux'

**Comment by danielgary** [comment]
Any chance you can unpublish this version or fix this issue in a new release?  This is a breaking change.  Either fix in a patch or unpublish and publish this release as a major version change?

We have a common library that depends on this and it took me an hour to track down where things went wrong.

**Comment by gor181** [comment]
Published a 1.1.6 with fix to this issue.
Should be fine.. apologies for the inconvenience..

**Comment by aurbano** [comment]
All working well, thanks!
